### PR TITLE
fix: extra space in link before punctuation

### PR DESCRIPTION
**Pull request by msdolbey**
Currently - there is an extra space that renders in the blogs if there is punctuation right after a link.

Example:
Screenshot 2023-09-05 at 2 21 08 PM

Found at the bottom of this post:
https://eng.d2iq.com/blog/profiling-kubernetes-controllers-with-pprof/

It appears it's because we added this:
https://github.com/mesosphere/d2iq-engineering-blog/blob/main/themes/casperv5/layouts/_default/_markup/render-link.html

This is a great add, because it will automatically make sure that links will go to a new tab/window. 
However, it adds an extra space if the link is followed by punctuation (`,` `.` etc.)
This probably wasn't found before, because the markdown will get rid of any double/extra spaces if it's just on a normal line, followed by other words. BUT, it looks like it adds it here because there's an extra line that's added at the bottom of this html file.

Getting rid of the new line gets rid of this nit picky sort of thing (from building it locally):
Screenshot 2023-09-05 at 2 21 32 PM

### CODE REVIEW

File: themes/casperv5/layouts/_default/_markup/render-link.html [hunk 1]
```diff
@@ -1 +1 @@
-<a href="{{ .Destination | safeURL }}"{{ with .Title }} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }} target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>
+<a href="{{ .Destination | safeURL }}"{{ with .Title }} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }} target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>
```
Comment (nataliepina) [inline themes/casperv5/layouts/_default/_markup/render-link.html:1]: What is the difference here that fixes it? My eyes cannot see it.

Comment (jimmidyson) [reply themes/casperv5/layouts/_default/_markup/render-link.html:1]: No new line at end of file which renders as whitespace

**Review by jimmidyson — APPROVED** [review]

File: themes/casperv5/layouts/_default/_markup/render-link.html (hunk 1 above)
Comment (nataliepina) [reply themes/casperv5/layouts/_default/_markup/render-link.html:1]: Ah I see now.

**Review by nataliepina — APPROVED** [review]
Thanks for fixing!

**Pull request merged**